docs/development/README.md

```diff
@@ -99,6 +99,23 @@ The cli binary will be available at `bin/p1` and can be used instead of `go run
 
 The commands available are listed [here](../../rpc/doc/README.md) or accessible via `bin/p1 --help`
 
+2.1 [OPTIONAL] Add the binary to your `.rc`
+
+You can add the following function so you can run the `p1` from anywhere on your host:
+
+```bash
+function p1 {
+    pocket_workdir="/Users/olshansky/workspace/pocket/pocket/"
+    if [ "$1" = "debug" ]; then
+        ${pocket_workdir}/bin/p1 debug --localhost=true --workdir="${pocket_workdir}"
```

Comment:
If we opt-in for utilizing environment variables here, here is how this part will look like (not tested though - this is an example):
```bash
        POCKET_WORKDIR = "${HOME}/workspace/pocket/pocket/"
        CONFIG_PATH = "${POCKET_WORKDIR}/build/config/config1.json"
        GENESIS_PATH = "${POCKET_WORKDIR}/build/config/genesis_localhost.json
        ${POCKET_WORKDIR}/bin/p1 debug
```